### PR TITLE
Add the Hacktoberfest Swag List repo

### PR DESCRIPTION
**Pull request by crweiner**
Hello fine folks at CodeSee and OSS Port! 👋 

I'm super happy to see you offering unique swag for Hacktoberfest and I'm happy to play my little part to help your project! I've also got an [Issue over here](https://github.com/crweiner/hacktoberfest-swag-list/issues/445) to add you to my Hacktoberfest Swag List project so we can send more meaningful PRs and contributors your way. 🎉 

Thank you for going the extra mile for Hacktoberfest! 🎃 🍂

### CODE REVIEW

File: projects/crweiner/hacktoberfest-swag-list.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+name: Hacktoberest Swag List
```
[the remark this answers is not in the document]
Comment (crweiner) [reply projects/crweiner/hacktoberfest-swag-list.mdx:2]: Ha!! 😆 You'd think I'd be able to spell the name of my own repo and favorite holiday! 😅 Thanks for catching that!

**Review by ramonh — APPROVED** [review]
Lovely! Thanks for fixing this so quick!

Welcome to Port 🚢

File: projects/crweiner/hacktoberfest-swag-list.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramonh) [reply projects/crweiner/hacktoberfest-swag-list.mdx]: No worries! Happens to the best of us! 💜

**Comment by crweiner** [comment]
Woohoo! Thanks for having me, @ramonh! I hope we can send some more great maintainers and contributors your way too once you're on the Swag List. 

Looks like we have similar overall goals. 👍 Thank you for supporting Hacktoberfest, making it easier to connect contributors to projects, and for building CodeSee so it's even simpler to dive into helping. You all rock! 😎

**Pull request merged**